src/libs/kata-sys-util/src/protection.rs

```diff
@@ -13,7 +13,7 @@ use std::path::Path;
 use std::path::PathBuf;
 use thiserror::Error;
 
-#[cfg(any(target_arch = "s390x", target_arch = "x86_64"))]
+#[cfg(any(target_arch = "s390x", target_arch = "powerpc64le"))]
```

Comment:
Out of curiosity, why powerpc64le was added here?

It seems not related to this commit.

Reply:
I've added a comment to the commit. Basically PPC64 needs this import fwics but it was previously missing.